conf/BuffCommand.conf.dist

```diff
@@ -11,5 +11,14 @@
 
 BuffCommand.Cooldown = 120
 
+# Settings to control the minimum level of .buff command
```

Comment:
I don't think it's necessary, it's already written in the `Description`, isn't it?